config/v1/types.go

```diff
@@ -5,6 +5,84 @@ import (
 	"k8s.io/apimachinery/pkg/runtime"
 )
 
+// +genclient
+// +genclient:nonNamespaced
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ImageConfig holds cluster-wide information about how to handle images.  The canonical name is `cluster`
+type ImageConfig struct {
+	metav1.TypeMeta `json:",inline"`
+	// Standard object's metadata.
+	metav1.ObjectMeta `json:"metadata,omitempty"`
+
+	// Format is the format of the name to be built for the system component
+	Format string `json:"format"`
+	// Latest determines if the latest tag will be pulled from the registry
+	Latest bool `json:"latest"`
+
+	// MaxImagesBulkImportedPerRepository controls the number of images that are imported when a user
+	// does a bulk import of a Docker repository. This number defaults to 50 to prevent users from
+	// importing large numbers of images accidentally. Set -1 for no limit.
+	MaxImagesBulkImportedPerRepository int32 `json:"maxImagesBulkImportedPerRepository"`
+	// DisableScheduledImport allows scheduled background import of images to be disabled.
+	DisableScheduledImport bool `json:"disableScheduledImport"`
+	// ScheduledImageImportMinimumIntervalSeconds is the minimum number of seconds that can elapse between when image streams
+	// scheduled for background import are checked against the upstream repository. The default value is 15 minutes.
+	ScheduledImageImportMinimumIntervalSeconds int32 `json:"scheduledImageImportMinimumIntervalSeconds"`
```

Comment:
This only used by the controller, but it feels more config-y somehow....

Reply:
I need to think about these as tuning.

Reply:
I do think these are things we might have to allow to be set by someone.  How would the tuning controller set them?

Reply:
If we don't have a tuning controller yet, would these go into spec?  Should we have to move something from spec to status or duplicate it?